lib/checks/index.js

```diff
@@ -11,9 +12,11 @@ module.exports = {
   canAddLikeTermPolynomialNodes,
   canMultiplyLikeTermConstantNodes,
   canMultiplyLikeTermPolynomialNodes,
+  canMultiplyNthRoots,
```

Comment:
Should this be `canMultiplyLikeTermNthRoots` to have consistent naming with the other files?